scripts/setup-operators.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
+set -x
+echo "Starting sealed secrets operator installation"
+install_sealed_secrets_operator(){
```

Comment:
Perhaps we should link to the documentation on this (or at least explain why) for folks who come along in the future and wonder why?

Reply:
Done